src/App.jsx

```diff
@@ -1,7 +1,24 @@
-export default function App() {
+import { AnimalListItem } from "./components/AnimalListItem";
+
+const App = () => {
```

Comment:
Your App.jsx would look more good if containing only the final component since you are using custom component!

i.e: you can have all of those functionalities in the animals component

Reply:
Actually, I went back into the document and I found it was also mentioned in the briefings of the exercises I just didn't it before

Thanks for the catch